conditional/blueprints/dashboard.py

```diff
@@ -58,7 +58,8 @@ def display_dashboard():
     c_meetings = [m.meeting_id for m in
                   MemberCommitteeAttendance.query.filter(
                       MemberCommitteeAttendance.uid == member.uid
-                  )]
+                  ) if CommitteeMeeting.query.filter(
```

Comment:
we have a bunch of oddities for SQL queries, but I think we could do a fancy statement with a single query.